src/libsyntax/print/pprust.rs

```diff
@@ -2270,6 +2270,11 @@ impl<'a> State<'a> {
                 self.print_expr(e)?;
                 word(&mut self.s, "?")?
             }
+            ast::ExprKind::Catch(ref blk) => {
+                self.head("catch")?;
```

Comment:
It's this 😄. You're seeing the "run the tests after putting them through pretty-printing" mode I bet.